core/templates/services/exploration-features-backend-api.service.ts

```diff
@@ -43,7 +43,7 @@ export class ExplorationFeaturesBackendApiService {
       private http: HttpClient,
       private urlInterpolationService: UrlInterpolationService) {}
 
-  fetchExplorationFeatures(
+  asyncfetchExplorationFeaturesAsync(
```

Comment:
Hm, this seems incorrect. Should it have been `async fetchExplorationFeaturesAsync(`?

Also, you will need to make changes in all the places this method is called.